working/augmentation-libraries/feature-specification.md

```diff
@@ -304,39 +303,39 @@ It is a compile-time error if:
     original declaration occurs, according to merge order. *An augmentation
     library can both declare a new declaration and augment it in the same file.*
 
-### Augment super
+### Augmented Expression
 
-The exact result of an `augment super` expression depends on what is being
+The exact result of an `augmented` expression depends on what is being
 augmented, but it follows generally the same rules as any normal identifier:
 
-*   **Augmenting getters**: Within an augmenting getter `augment super` invokes
-    the getter and evaluates to the return value. If augmenting a field with a
+*   **Augmenting getters**: Within an augmenting getter `augmented` invokes the
+    getter and evaluates to the return value. If augmenting a field with a
```

Comment:
Probably worth noting here and in the other bullets that if the augmented member is an instance member, it is invoked with the same value for `this`.

(Sorry, I didn't make this whole section very precise when I first wrote it and it definitely needs some tightening up. 🙃 )